getting-started/generative-ai.rst

```diff
@@ -0,0 +1,25 @@
+.. _generative-ai:
+
+=============
+Generative AI
+=============
+
+Generative AI has evolved rapidly over the past decade and will continue in the future.
+Using Generative AI and LLMs (Large Language Models) can be helpful tools for contributors.
+Their overuse can also be problematic, such as generation of incorrect code, inaccurate documentation, and unneeded code churn.
+Discretion, good judgement, and critical thinking **must** be used when opening issues and pull requests.
+
+Acceptable uses
+===============
+
+Some of the acceptable uses of generative AI include:
+
+- Assistance with writing comments, especially in a non-native Language
+- Gaining understanding of existing code
+- Supplementing contributor knowledge for code, tests, and documentation
+
+Unacceptable uses
+=================
+
+If a maintainer believes that a PR or issue was **fully generated** by AI tools, the PR or issue will be closed.
```

Comment:
I'd prefer to cast this as a more general "Issues or PRs that are not productive will be closed, including those that are AI-generated." Focus on the fact the issue is problematic, not on how the issue got there.

Reply:
@JelleZijlstra Do you want to suggest wording? That makes sense to me.